src/MSBuildLocator/Microsoft.Build.Locator.csproj

```diff
@@ -11,6 +11,7 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>key.snk</AssemblyOriginatorKeyFile>
     <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
+	<RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
This makes it work when building locally (`NETCoreSdkRuntimeIdentifier` is what the SDK is running on) but it's probably not enough to build a package that can be used everywhere.

Can we easily enumerate the problematic RIDs and package up the native dependency for all of them?

Reply:
I wasn't able to enumerate them easily, but listing runtime assemblies works as expected